examples/defi/contract-deployment.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Example: Deploy a Clarity smart contract programmatically
+ * 
+ * This example shows how to deploy a contract to Stacks blockchain
+ * using stacks.js libraries.
+ */
+
+import {
+  makeContractDeploy,
+  broadcastTransaction,
+  AnchorMode,
+  PostConditionMode,
+} from '@stacks/transactions';
+import { StacksMainnet, StacksTestnet } from '@stacks/network';
+
+// Contract source code (simple counter example)
```

Comment:
Simple counter is not defi and not real world example as mentioned in the readme.